addons/binding/org.openhab.binding.kodi/README.md

```diff
@@ -68,11 +68,28 @@ The Kodi thing supports the following channels:
 | album                   | String       | Album name of the currently played song |
 | artist                  | String       | Artist name of the currently played song or director of the currently played movie|
 | playuri                 | String       | Plays the file with the provided URI |
+| pvropentv               | String       | Opens the PVR TV channel with the provided name |
+| pvropenradio            | String       | Opens the PVR Radio channel with the provided name |
 | shownotification        | String       | Shows the provided notification message on the screen |
 | input                   | String       | Allows to control Kodi. Valid values are: `Up`, `Down`, `Left`, `Right`, `Select`, `Back`, `Home`, `ContextMenu`, `Info`, `ShowCodec`, `ShowOSD` |
 | inputtext               | String       | This channel emulates a keyboard input |
 | systemcommand           | String       | This channel allows to send commands to `shutdown`, `suspend`, `hibernate`, `reboot` kodi |
-| mediatype               | String       | The media type of the current file. e.g. song or movie | 
+| mediatype               | String       | The media type of the current file. Valid return values are: `unknown`, `channel`, `episode`, `movie`, `musicvideo`, `picture`, `radio`, `song`, `video` |
+
+### Channel Configuration
+
+**group** The PVR channels can be put into userdefined PVR channel groups. There are two default PVR channel groups. One for PVR TV channels and one for PVR radio channels. The default labels are "All channels" (in german systems "Alle Kanäle"). You have to adjust this configuration to use the `pvropentv` and `pvropenradio` channels properly. You can optionally configure an userdefined PVR channel group.
```

Comment:
I changed the spelling and commited it in new PR [#2084](https://github.com/openhab/openhab2-addons/pull/2084). Sry for the circumstances.